Makefile

```diff
@@ -91,10 +91,59 @@ ifdef SELINUX_ENABLED
 ifeq ($(shell test "$(SELINUX_ENABLED)" = True  -o  \
 									 "$(SELINUX_ENABLED)" = true && printf "true"), true)
 		@$(eval export SELINUX_ENABLED=,Z)
+else
+		@$(eval export SELINUX_ENABLED='')
 endif
 endif
 	@echo
 
+ifndef IMGTAG
+	@$(eval export IMGTAG=localhost/kubevirt-user-guide)
+else
+ifeq ($(shell test $IMGTAG > /dev/null 2>&1 && printf "true"), true)
+	@echo WARN: Using IMGTAG=$$IMGTAG
+	@echo
+else
+	@$(eval export IMGTAG=localhost/kubevirt-user-guide)
+endif
+endif
+
+
+## Build site. This target should only be used by Netlify and Prow
+build: envvar
+	@echo "${GREEN}Makefile: Build mkdocs site${RESET}"
+	which $(PYTHON)
+	$(PYTHON) -m venv /tmp/venv
+	. /tmp/venv/bin/activate
+	$(PIP) install mkdocs mkdocs-awesome-pages-plugin mkdocs-htmlproofer-plugin
+	@echo
+	@echo '*** BEGIN cat mkdocs.yml ***'
+	@cat mkdocs.yml
+	@echo '*** END cat mkdocs.yml ***'
+	mkdocs build -f mkdocs.yml -d site
+
+
+## Build image localhost/kubevirt-user-guide
+build_img: | envvar
+	@echo "${GREEN}Makefile: Building Image ${RESET}"
+	${DEBUG}if [ ! -e "./Dockerfile" ]; then \
+	  IMAGE="`echo $${IMGTAG} | sed -e s#\'##g -e s#localhost\/## -e s#:latest##`";  \
+	  if [ "`curl https://raw.githubusercontent.com/kubevirt/project-infra/master/images/kubevirt-user-guide/Dockerfile -o ./Dockerfile -w '%{http_code}\n' -s`" != "200" ]; then \
```

Comment:
Why not use `--fail` here?

Reply:
Excellent idea!  I also modified the other curl as well as the curls in website Makefile